src/main/java/com/idrsolutions/microservice/utils/DefaultFileServlet.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 IDRSolutions
+ * Copyright 2022 IDRsolutions
  * Copyright 2021 OmniFaces
  *
```

Comment:
@leonatherton Do we need to update the OmniFaces year as well for does this need to remain at 2021?

Reply:
@KieranFrance I would leave it as-is so as to not modify the original copyright notice.
https://fossa.com/blog/open-source-licenses-101-apache-license-2-0/